playwright/tests/e2e/steps/transactions-overview.steps.ts

```diff
@@ -0,0 +1,221 @@
+});
+When(
+  /^the user selects the transaction hash of one record of the transactions table in the transactions page$/,
+  async ({ page }) => {
+    const transactionsTableValueEpoch = page.getByTestId("transaction.table.value.txhash#0");
+    transactionHash = await transactionsTableValueEpoch.getAttribute("href");
+    await transactionsTableValueEpoch.click();
+  }
+);
+Then(/^the transaction details page of the select transaction should be opened$/, async ({ page }) => {
+  await transactionDetailPage(page).checkTransactionsDetail({ transactionHash: transactionHash || "" });
+});
+
+Given(/^the user is in the Transactions page portal$/, async ({ page }) => {
+  await transactionOverviewPage(page).goToTransactions();
+});
+When(
+  /^the user selects the block number of one record of the transactions table in the transactions page$/,
+  async ({ page }) => {
+    const transactionsTableValueEpoch = page.getByTestId("transaction.table.value.block#0");
+    await transactionsTableValueEpoch.click();
+  }
+);
+Then(/^the user should be redirected to the block detail page of the selected block number$/, async ({ page }) => {
+  const deatailPageTitle = page.getByTestId("detail.page.title");
+  await expect(deatailPageTitle, "Check title on block detail").toHaveText("Block Details");
+});
+
+Given(/^the user is in the Transactions page web$/, async ({ page }) => {
+  await transactionOverviewPage(page).goToTransactions();
+});
+When(
+  /^the user selects the epoch number of one record of the transactions table in the transactions page$/,
+  async ({ page }) => {
+    const transactionsTableValueEpoch = page.getByTestId("transactions.table.value.epoch#0");
+    await transactionsTableValueEpoch.click();
+  }
+);
+Then(/^the user should be redirected to the epoch detail page of the selected epoch number$/, async ({ page }) => {
+  const deatailPageTitle = page.getByTestId("detail.page.title");
+  await expect(deatailPageTitle, "Check title on epoch detail").toHaveText("Epoch Details");
+});
+
+Given(/^the user is in the Transactions page for go to address detail page$/, async ({ page }) => {
+  await transactionOverviewPage(page).goToTransactions();
+});
+When(
+  /^the user selects the input or output address of one record of the transactions table in the transactions page$/,
+  async ({ page }) => {
+    const transactionsTableValueEpoch = page.getByTestId("transaction.table.value.inputAddress#0").first();
+    addressId = await transactionsTableValueEpoch.getAttribute("href");
+    await transactionsTableValueEpoch.click();
+  }
+);
+Then(/^the user should be redirected to the address details page of the selected address hash$/, async ({ page }) => {
+  await transactionDetailPage(page).checkAddressDetail({ addressId: addressId || "" });
+});
+
+Then(/^the user should be redirected to the address detail page of the selected address hash$/, async ({ page }) => {
+  const deatailPageTitle = page.getByTestId("detail.page.title");
```

Comment:
This validation should be executed in the epoch detail page class and here just call the method that executes the validation.